doc/sphinx/proofs/writing-proofs/reasoning-inductives.rst

```diff
@@ -974,32 +974,42 @@ Generation of induction principles with ``Scheme``
    .. insertprodn scheme_kind sort_family
 
    .. prodn::
-      scheme_kind ::= Equality for @reference
-      | {| Induction | Minimality | Elimination | Case } for @reference Sort @sort_family
+      scheme_kind ::= {| Induction | Minimality | Elimination | Case } for @reference Sort @sort_family
       sort_family ::= Set
       | Prop
       | SProp
       | Type
 
-  A high-level tool for automatically generating
-  (possibly mutual) :term:`induction principles <induction principle>`
-  for given types and sorts.
-  Each :n:`@reference` is a different inductive type identifier belonging to
-  the same package of mutual inductive definitions.
-  The command generates the :n:`@ident`\s as mutually recursive
-  definitions. Each term :n:`@ident` proves a general principle of mutual
-  induction for objects in type :n:`@reference`.
+   Generates (possibly mutual) :term:`induction principles <induction principle>`
+   for given types and sorts. Each :n:`@reference` is a different inductive type
+   identifier belonging to the same package of mutual inductive definitions. The
+   command generates the :n:`@ident`\s as mutually recursive definitions. Each
+   term :n:`@ident` proves a general principle of mutual induction for objects
+   in type :n:`@reference`.
 
-  :n:`@ident`
-    The name of the scheme. If not provided, the scheme name will be determined automatically
-    from the sorts involved.
+   :n:`@ident`
+      The name of the scheme. If not provided, the scheme name will be determined
+      automatically from the sorts involved.
+
+   The following :n:`@scheme_kind`\s generate induction principles with
+   given properties:
+
+   =================== =========== ===========
+    :n:`@scheme_kind`   Dependent   Recursive
+   =================== =========== ===========
+    :n:`Induction`         Yes         Yes
+    :n:`Minimality`        No          Yes
+    :n:`Elimination`       Yes         No
+    :n:`Case`              No          No
+   =================== =========== ===========
+
+   See examples :ref:`here <scheme_example>`.
 
-  :n:`Minimality for @reference Sort @sort_family`
-    Defines a non-dependent elimination principle more natural for inductively defined relations.
+.. cmd:: Scheme Equality for @reference
```

Comment:
```suggestion
.. cmd:: Scheme Equality for @reference
   :name: Scheme Equality
```

Will make the name in the index be "Scheme Equality" instead of "Scheme Equality for".  FWIW, "for" in the grammar doesn't add much except to make the syntax more consistent with "Scheme Induction"--where it also doesn't add much, either.

Reply:
If I change the "for" I will have to write (non-backwards compatible) overlays for all the uses of `Scheme Equality`. I think it is better to keep it like this. I will include "for" in the index however.

Reply:
I think I'd put this below the 3 examples of `Scheme`.  Also the first two example should be indented like the third one (visually associating them with `Scheme`.

Reply:
Will remove the for tho